apollo-client/src/main/java/com/ctrip/framework/apollo/internals/DefaultConfig.java

```diff
@@ -29,7 +29,8 @@
  * @author Jason Song(song_s@ctrip.com)
  */
 public class DefaultConfig extends AbstractConfig implements RepositoryChangeListener {
-  private static final Logger logger = LoggerFactory.getLogger(DefaultConfig.class);
+
+  private static final Logger logger = DeferredLoggerFactory.getLogger(DefaultConfig.class);
```

Comment:
I think `com.ctrip.framework.apollo.internals.AbstractConfigFile` also need to use `DeferredLogger`

Reply:
yes，It is